config/initializers/sidekiq.rb

```diff
@@ -0,0 +1,7 @@
+Sidekiq.configure_server do |config|
+  config.redis = { url: 'redis://localhost:6379/' }
```

Comment:
You can put `redis://localhost:6379/` into env variables 
It will give you the ability to use different URLs depending on the environment

Reply:
Read more here https://railshero.pw/489/all-about-heroku-environment-variables/ if you haven't use env variables yet